lib/src/resolvers/cluster.ts

```diff
@@ -43,10 +43,18 @@ export async function getTopClusters(
   { dataSources }: ContextValue,
 ): Promise<Cluster[]> {
   const { first, after, filter } = getQueryParams(params);
-  const { mintableBy } = filter ?? {};
+  const { mintableBy, codeHash } = filter ?? {};
```

Comment:
"codeHash" was not defined in the "Query.topClusters" in the schema. 
You should add it to the "TopClusterFilterInput" type.